config/interpolate_funcs_test.go

```diff
@@ -174,76 +175,52 @@ func TestInterpolateFuncCoalesce(t *testing.T) {
 	})
 }
 
-func TestInterpolateFuncDeprecatedConcat(t *testing.T) {
```

Comment:
Ah okay - I was thinking of [this line](https://github.com/hashicorp/terraform/pull/6011/files/a4caaaced273f1cdaa5593997c9ca04d526de5d4#diff-38c6c95b4ed4e10a18cdf045d73d94dbR206) which perhaps is something different.